server/src/main/java/com/jungleapp/cs414/server/JungleBoard.java

```diff
@@ -70,10 +66,6 @@ private boolean validPositionString(String position) {
 
         int rowPos = position.charAt(0)-48;
         int colPos = position.charAt(1)-48;
-        if(rowPos > 8 || rowPos < 0 || colPos < 0 || colPos > 6) {
-            return false;
-        }
-
-        return true;
+        return rowPos <= 8 && rowPos >= 0 && colPos >= 0 && colPos <= 6;
```

Comment:
I believe this will return out of bounds exceptions for the upper limits of the board. When we try to make a move **row + 1** the condition of **rowPos <= 8** will pass even if we are on the edge of the board, and the program will add **row = row + 1** i.e **row == 9**, which is OOB.

Reply:
Yeah, you're right. I merged this from your initial branch and didn't even realize that change. I say we still merge this one and then update that part in a future merge since it is so small.